README.md

```diff
@@ -1 +1,70 @@
-# Book-Store
\ No newline at end of file
+# Getting Started with Create React App
+
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+
+## Available Scripts
```

Comment:
- [ ] Your readme isn't descriptive, and it doesn't provide enough details on the project. Kindly delete the boilerplate that got created by the CRA, and include some project information. You may use the [Microver template](https://github.com/microverseinc/readme-template) as an example.